docs/jotai/basics/async.mdx

```diff
@@ -73,3 +73,41 @@ const Component = () => {
 ```
 
 > There's no way to know as of now if an atom suspends because of `read` or `write`.
+
+## Triggering `Suspense` fallback
```

Comment:
```suggestion
### Triggering `Suspense` fallback
```

This section applies only for "Async write atom" not "Async read atom" which works totally different with Suspense.

Reply:
:) 

Oh yes totally I'll update this now.

Reply:
Changes comitted.